hw2/05-form.html

```diff
@@ -1,20 +1,141 @@
-    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <title>05 - Form</title>
-    <link
-      rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
-    <script defer src="05-form.js"></script>
-  </head>
-  <body>
-    <main>
-      <form>
-        <h1>Exercise 05 - Form</h1>
-      </form>
-    </main>
-  </body>
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+        <title>05 - Form</title>
+        <link
+            rel="stylesheet"
+            href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+        />
+        <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+        <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"></script>
+        <script defer src="05-form.js"></script>
+    </head>
+    <body class="bg-dark">
+        <main class="col-6 bg-light container mt-5 rounded shadow ps-3">
+            <h1 class="text-center fw-bold pt-3 pb-2">Exercise 05 - Form</h1>
+            <form id="dataForm" class="py-3 w-100">
+                <label class="w-100 fw-bold pb-1 rounded-3" for="fullName"
+                    >Full Name:
+                </label>
+                <div>
+                    <input
+                        class="w-100 form-control"
+                        id="fullName"
+                        type="name"
+                        name="fullName"
+                    />
+                </div>
+                <label class="w-100 fw-bold pb-1 pt-3 rounded-3" for="email"
+                    >Email:
+                </label>
+                <div>
+                    <input
+                        class="w-100 form-control"
+                        id="email"
+                        type="email"
+                        name="email"
+                    />
+                </div>
+                <label
+                    class="w-100 fw-bold pb-1 pt-3 rounded-3"
+                    for="registrationStatus"
+                    >Registration Status:
+                </label>
+                <div>
+                    <select
+                        class="w-100 form-select"
+                        id="registrationStatus"
+                        name="registrationStatus"
+                    >
+                        <option value="" hidden selected>
+                            Choose an Option
+                        </option>
+                        <option value="Registered">Registered</option>
+                        <option value="Withdrawn">Withdrawn</option>
+                    </select>
+                    <div class="fw-bold pb-1 pt-3 rounded-3">
+                        Have you taken any of the following courses?
+                    </div>
+                    <div>
```

Comment:
Consider adding a fieldset around your checkboxes to add more semantical structure for accessibility.